### ISSUE TITLE
Person[s]Name Swaps in Multiple Face Detection using classifier_webcam.py

### ISSUE THREAD

**Issue by BhagyodayPatil** [cmusatyalab/openface#284]
### Context of the issue.
I have created a classifier.pkl file with around 10 people(20 images each) using the First 4 Steps given under DEMO 3: Training a Classifier.
When i run the classifier_webcam.py for the same file (i.e custom classifier.pkl as args) the window frame opens and starts detection.
When a Single person is facing the webcam the detection is accurate with maximum confidence.
When Multiple people are facing the webcam the detection is not consistent on the names of the persons[], it swaps the persons[] name quite often.
Also when a Unknown & Identified(face in classifier.pkl) is facing the webcam the same thing happens, it swaps the name of the persons[]. This name swapping is quite often as well. 

### Expected behaviour.
Whenever Multiple Faces are facing the Webcam, the identified names should be consistent & not keep changing every now and then.
Whenever a Unknown & Identified face faces the webcam, the names displayed should be consistent & not swap with one or the other persons[] name.

### Actual behaviour.
When Multiple people are facing the webcam the detection is not consistent on the names of the persons[], it swaps the persons[] name quite often.
Also when a Unknown & Identified(face in classifier.pkl) is facing the webcam the same thing happens, it swaps the name of the persons[]. This name swapping is quite often as well. 

### Steps to reproduce.
* Take 20 images each of 10 people.
* Save it folder wise with their names & hierarchy (as mentioned in DEMO 3 Step 1: Creating Raw Image Directory)
* Use align-dlib.py to preprocess the raw images (as mentioned in DEMO 3 Step 2: Preprocess the raw Images)
* Generate representations to create reps.csv & labels.csv (as mentioned in DEMO 3 Step 3: Generate Representations)
* Create classification model i.e classifier.pkl file (as mentioned in DEMO 3 Step 4: Create the classification Model)
* Finally run the ./demos/classifier_webcam.py with the generated classifier.pkl file as argument 
* Try it with the same 10 people one by one 
* Try it with multiple people from the same 10
* Try it with one from the 10 & one unknown(not from the 10)
* Observe that the names appearing in the numpy array aren't constant if multiple faces are the input for the camera feed
* Observe that names appearing in the numpy array aren't constant if a Known & Unknown face[s] are the input for the camera feed

### OS and hardware information.

+ Operating system: [Ubuntu 16.04 LTS]
+ Torch version: [Torch7]
+ CPU architecture: [Intel(R) Pentium(R) CPU G3260 @ 3.30GHz Dual Core] [Haskwell]
+ GPU type (if using): [Integrated Graphics: Intel HD]
+ WEBCAM : Logitech C930E 

FYI: [http://www.cpu-world.com/CPUs/Pentium_Dual-Core/Intel-Pentium%20G3260.html]

**Comment by deepmore1** [comment]
i think swapping is not a real problem, after all, if you do as in web demo where you assign to each face a bounding box and it's own prediction and u get a false prediction ,there you will be sure you have a problem! keep me in touch if you got anything i'm working on classifier_webcam as well. good luck

**Comment by BhagyodayPatil** [comment]
I didn't exactly understand what you're saying about Web Demo1. I have only used Steps in Demo 3.
Also I want to recognise Multiple People simultaneously(with rectangle[s] around their Faces), not one at a Time. 
PFA demo video of what exactly we are working on & where exactly the name swap occurs.
As I am new to this Topic, I am unable to make out where exactly is the problem?
Any Help is always appreciated.
@deepmore1 @bamos 

Confidence Threshold is 0.6 in Demo, below 0.6 will be Unknown
[Multiface Focus Demo.mp4.zip](https://github.com/cmusatyalab/openface/files/1094051/Multiface.Focus.Demo.mp4.zip)

Referring to #5 I tried it, but its focuses(shows rectangle) only on One Face at a Time, multiple faces simultaneously isn't working with this solution.
